src/NECompletion/CompletionContext.class.st

```diff
@@ -10,11 +10,10 @@ Class {
 		'completionToken',
 		'ast',
 		'node',
-		'isWorkspace',
 		'class',
 		'entries',
-		'editor',
-		'sorter'
+		'sorter',
+		'engine'
 	],
```

Comment:
- `isWorkspace` can be asked to the engine: remove it
- the `editor` should not be directly modified from the completion context, otherwise it generates a dependency to morphic. Do it by talking to the `engine` directly which will hide that.